bot/exts/easter/earth_photos.py

```diff
@@ -0,0 +1,48 @@
+import discord
+from discord.ext import commands
+
+from bot.constants import Colours
+from bot.constants import Tokens
+
+UnClient_id = Tokens.unsplash_key
+
+
+class EarthPhotos(commands.Cog):
+    """This cog contains the command for earth photos."""
+
+    def __init__(self, bot: commands.Bot):
+        self.bot = bot
+
+    @commands.command(aliases=["earth"])
+    async def earth_photos(self, ctx: commands.Context) -> None:
+        """Returns a random photo of earth, sourced from Unsplash."""
+        async with ctx.typing():
+            async with self.bot.http_session.get(
+                    'https://api.unsplash.com/photos/random',
+                    params={"query": "earth", "client_id": UnClient_id}) as r:
+                jsondata = await r.json()
+                linksdata = jsondata.get("urls")
+                embedlink = linksdata.get("regular")
+                downloadlinksdata = jsondata.get("links")
+                userdata = jsondata.get("user")
+                username = userdata.get("name")
+                userlinks = userdata.get("links")
+                profile = userlinks.get("html")
+            async with self.bot.http_session.get(
+                downloadlinksdata.get("download_location"),
+                    params={"client_id": UnClient_id}
+            ) as _:
+                pass
+            embed = discord.Embed(
+                title="Earth Photo",
+                description="A photo of earth from Unsplash.",
+                color=Colours.grass_green
+            )
+            embed.set_image(url=embedlink)
+            embed.add_field(name="Author", value=f"Made by [{username}]({profile}) on Unsplash.")
+        await ctx.send(embed=embed)
+
+
+def setup(bot: commands.Bot) -> None:
```

Comment:
Could you add a check here, so that if the token isn't set the cog doesn't try to load. As it raises an error currently if the token isn't present.
For example from the games cog:
https://github.com/python-discord/sir-lancebot/blob/a3bd208580ff223cd0120d0c4b5a1bf3c28d03e5/bot/exts/evergreen/game.py#L472-L481

Reply:
sure!